website/cue/reference/components.cue

```diff
@@ -439,11 +443,26 @@ components: {
 		default_namespace: string
 	}
 
-	#Output: {
+	#OutputData: {
 		logs?:    #LogOutput
 		metrics?: #MetricOutput
 	}
 
+	#Output: {
+		name:        string
+		description: string
+	}
+
+	#Outputs: {
+		_default_output: #Output & {
+			name:        ""
+			description: "Use this component's ID as an input to downstream transforms and sinks."
```

Comment:
Maybe we could also say something like "Default output stream of the component."? Right now its clear how to reference it, but not what it is.